packages/docusaurus-plugin-content-blog/src/__tests__/__snapshots__/feed.test.ts.snap

```diff
@@ -50,6 +50,14 @@ exports[`atom has feed item for each post 1`] = `
             <uri>https://sebastienlorber.com</uri>
         </author>
     </entry>
+    <entry>
+        <title type="html"><![CDATA[unlisted]]></title>
+        <id>/unlisted</id>
+        <link href="https://docusaurus.io/myBaseUrl/blog/unlisted"/>
+        <updated>2020-02-27T00:00:00.000Z</updated>
+        <summary type="html"><![CDATA[this post is unlisted]]></summary>
+        <content type="html"><![CDATA[<p>this post is unlisted</p>]]></content>
+    </entry>
```

Comment:
looks like a mistake: blog post is not filtered from the blog feeds.

See https://deploy-preview-8004--docusaurus-2.netlify.app/tests/blog/rss.xml